lib/mkl/libonemkl.jl

```diff
@@ -42,6 +43,23 @@ function onemklZgemm(device_queue, transA, transB, m, n, k, alpha, A, lda, B, ld
                                     C::ZePtr{ComplexF64}, ldc::Int64)::Cint
 end
 
+function onemklDnrm2(device_queue, ctx, dev, n, x, incx, result)
+	@ccall liboneapi_support.onemklDnrm2(device_queue::syclQueue_t, ctx::syclContext_t, dev::syclDevice_t, n::Int64, x::ZePtr{Cdouble}, incx::Int64, result::RefOrZeRef{Cdouble})::Cvoid
+end
+
+function onemklSnrm2(device_queue, ctx, dev, n, x, incx, result)
+	@ccall liboneapi_support.onemklSnrm2(device_queue::syclQueue_t, ctx::syclContext_t, dev::syclDevice_t, n::Int64, x::ZePtr{Cfloat}, incx::Int64, result::RefOrZeRef{Cfloat})::Cvoid
+end
+
+function onemklCnrm2(device_queue, ctx, dev, n, x, incx, result)
+	@ccall liboneapi_support.onemklCnrm2(device_queue::syclQueue_t, ctx::syclContext_t, dev::syclDevice_t, n::Int64, x::ZePtr{ComplexF32}, incx::Int64, result::RefOrZeRef{Cfloat})::Cvoid
+end
+
+function onemklZnrm2(device_queue, ctx, dev, n, x, incx, result)
+	@ccall liboneapi_support.onemklZnrm2(device_queue::syclQueue_t, ctx::syclContext_t, dev::syclDevice_t, n::Int64, x::ZePtr{ComplexF64}, incx::Int64, result::RefOrZeRef{Cdouble})::Cvoid
```

Comment:
Would be great if you can split long lines, it improves readability.

Reply:
Done